OwnTube.tv/App.tsx

```diff
@@ -1,12 +1,14 @@
 import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
+import { StyleSheet, Text, View, Image } from "react-native";
 import VideoDataService from "./components/videosOverview";
 
 import build_info from "./build-info.json";
 
 export default function App() {
   return (
     <View style={styles.container}>
+      {/* Using the SVG component */}
+      <Image source={require("./assets/Logo300x100.png")} />
```

Comment:
Would be good the define export const Logo in the assets, so you can use it anywhere. 
Size of the image is not important do define IMHO.